packages/components/src/ListingSummary.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from "react";
+import styled, { StyledComponentClass } from "styled-components";
+import { EthAddress } from "@joincivil/core";
+import { colors, fonts } from "./styleConstants";
+import { SectionHeading } from "./Heading";
+import { buttonSizes, InvertedButton } from "./Button";
+import { CountdownTimer } from "./PhaseCountdown";
+import {
+  AwaitingApprovalStatusLabel,
+  CommitVoteStatusLabel,
+  RevealVoteStatusLabel,
+  RequestingAppealStatusLabel,
+} from "./ApplicationPhaseStatusLabels";
+
+export const StyledListingSummaryList = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  width: 1200px;
+`;
+
+const StyledListingSummaryContainer = styled.div`
+  border: 1px solid ${colors.accent.CIVIL_GRAY_4};
+  box-shadow: inset 0 1px 0 0 ${colors.accent.CIVIL_GRAY_4}, 0 2px 4px 0 ${colors.accent.CIVIL_GRAY_3};
+  box-sizing: border-box;
+  height: 491px;
+  margin: 0 30px 48px 0;
+  width: 379px;
+
+  &:nth-child(3n + 3) {
+    margin-right: 0;
+  }
+`;
+
+const StyledListingSummaryNewsroomName = SectionHeading.extend`
+  margin: 0 0 16px;
+`;
+
+const StyledListingSummaryHed = styled.div`
+  display: flex;
+  padding: 27px 23px 30px;
+`;
+
+const StyledListingSummaryDek = styled.div`
+  border-top: 1px solid ${colors.accent.CIVIL_GRAY_4};
+  color: ${colors.primary.CIVIL_GRAY_1};
+  font: normal 18px/33px ${fonts.SANS_SERIF};
+  padding: 27px 23px 30px;
+`;
+
+const NewsroomIcon = styled.figure`
+  background: ${colors.accent.CIVIL_GRAY_4};
+  margin: 0 17px 0 0;
+  height: 80px;
+  min-width: 80px;
+`;
+
+const MetaItem = styled.div`
+  margin: 0 0 16px;
+`;
+const MetaLabel = styled.div`
+  color: ${colors.primary.CIVIL_GRAY_1};
+  font: normal 16px/16px ${fonts.SANS_SERIF};
+  margin: 0 0 6px;
+`;
+const MetaValue = styled.abbr`
+  color: ${colors.primary.CIVIL_GRAY_1};
+  display: block;
+  font: normal 14px/17px ${fonts.SANS_SERIF};
+  max-width: 65%;
+  overflow-y: hidden;
+  text-decoration: none;
+  text-overflow: ellipsis;
+`;
+
+export interface ListingSummaryComponentProps {
+  address?: EthAddress;
```

Comment:
do you think it's better to pass all these props in individually instead of just pulling them out from a ListingWrapper / Newsroomwrapper in render()?

Reply:
I went back and forth about this, but I think this props structure for these very view-specific components would be good for the future.

I think we could eventually have the design team manage these components / styling / etc, and having the components ignorant of the `ListingWrapper` / `NewsroomWrapper` makes them easier to work on in Storybook and apply designs to.

I think we ideally have either a selector layer added to our Redux setup that creates these flat structures for the dumb components... or a shared method that does the translation.